docs/docsite/rst/playbook_guide/playbooks_handlers.rst

```diff
@@ -114,6 +114,39 @@ Each handler should have a globally unique name. If multiple handlers are define
 
 There is only one global scope for handlers (handler names and listen topics) regardless of where the handlers are defined. This also includes handlers defined in roles.
 
+Controlling the notifing of handlers by defining "changed"
+----------------------------------------------------------
+
+By using ``changed_when`` keyword, you define when a task has "changed" status. This means you directly determine when ``notify`` is triggered so it executes the handlers.
```

Comment:
I would combine these two sentences and maybe shorten them a little. Consider:

You can control when handlers are notified about task changes using the `changed_when` keyword.

This puts the focus on the more relevant details to this part of the docs, namely controlling when handlers get notified about changes. In the original first sentence it reads more like documentation about the `changed_when` keyword itself.